src/common/config.cc

```diff
@@ -604,15 +604,19 @@ int md_config_t::parse_option(std::vector<const char*>& args,
 
   if (ret != 0 || !error_message.empty()) {
     if (oss) {
-      *oss << "Parse error setting " << option_name << " to '"
-           << val << "' using injectargs";
+      if(option_name && !val.empty()) {
```

Comment:
to silence the warning is the side effect, the point is to improve the quality of code, not to appease the static analyzer. if this does not happen, we should `assert()` it.